generated/nirfmxwlan/nirfmxwlan.proto

```diff
@@ -31,27 +590,3421 @@ enum Boolean {
+}
+
+message DSSSModAccCfgMeasurementLengthResponse {
+  int32 status = 1;
+}
+
+message DSSSModAccCfgPowerMeasurementCustomGateArrayRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  repeated double start_time = 3;
+  repeated double stop_time = 4;
+}
+
+message DSSSModAccCfgPowerMeasurementCustomGateArrayResponse {
+  int32 status = 1;
+}
+
+message DSSSModAccCfgPowerMeasurementEnabledRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof power_measurement_enabled_enum {
+    DsssModAccPowerMeasurementEnabled power_measurement_enabled = 3;
+    int32 power_measurement_enabled_raw = 4;
+  }
+}
+
+message DSSSModAccCfgPowerMeasurementEnabledResponse {
+  int32 status = 1;
+}
+
+message DSSSModAccCfgPowerMeasurementNumberOfCustomGatesRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  int32 number_of_custom_gates = 3;
+}
+
+message DSSSModAccCfgPowerMeasurementNumberOfCustomGatesResponse {
+  int32 status = 1;
+}
+
+message DSSSModAccFetchAveragePowersRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message DSSSModAccFetchAveragePowersResponse {
+  int32 status = 1;
+  double preamble_average_power_mean = 2;
+  double header_average_power_mean = 3;
+  double data_average_power_mean = 4;
+  double ppdu_average_power_mean = 5;
+}
+
+message DSSSModAccFetchConstellationTraceRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message DSSSModAccFetchConstellationTraceResponse {
+  int32 status = 1;
+  repeated nidevice_grpc.NIComplexNumberF32 constellation = 2;
+  int32 actual_array_size = 3;
+}
+
+message DSSSModAccFetchCustomGatePowersArrayRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message DSSSModAccFetchCustomGatePowersArrayResponse {
+  int32 status = 1;
+  repeated double average_power_mean = 2;
+  repeated double peak_power_maximum = 3;
+  int32 actual_array_size = 4;
+}
+
+message DSSSModAccFetchDecodedHeaderBitsTraceRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message DSSSModAccFetchDecodedHeaderBitsTraceResponse {
+  int32 status = 1;
+  repeated int32 decoded_header_bits = 2;
+  int32 actual_array_size = 3;
+}
+
+message DSSSModAccFetchDecodedPSDUBitsTraceRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message DSSSModAccFetchDecodedPSDUBitsTraceResponse {
+  int32 status = 1;
+  repeated int32 decoded_psdu_bits = 2;
+  int32 actual_array_size = 3;
+}
+
+message DSSSModAccFetchEVMRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message DSSSModAccFetchEVMResponse {
+  int32 status = 1;
+  double rmsevm_mean = 2;
+  double peak_evm80211_2016_maximum = 3;
```

Comment:
Note: not part of this change.

We should track these as CONSIDER changing to `peak_evm_80211_2016_maximum`. The default doesn't tokenize for numbers (some works as their own token and some don't).